app/views/pinned_links/index.html.erb

```diff
@@ -0,0 +1,81 @@
+<h1>Featured Links</h1>
+<p class="is-lead">Featured links allow you as a moderator to draw attention to issues or posts of importance to the community. Use them sparingly and only when needed, because they tend to get ignored if used too often.</p>
+
+<div class="grid has-margin-bottom-4">
+    <% if current_user.is_global_moderator %>
+    <div class="grid--cell">
+        <div class="button-list is-gutterless">
+            <%= link_to "current community", "?global=0&filter="+(params[:filter]||'active'), class: "button is-muted is-outlined #{(params[:global]=='0' || params[:global].nil?) ? 'is-active' : ''}" %>
+            <%= link_to "global", "?global=1&filter="+(params[:filter]||'active'), class: "button is-muted is-outlined #{(params[:global]=='1') ? 'is-active' : ''}" %>
+            <%= link_to "everywhere", "?global=2&filter="+(params[:filter]||'active'), class: "button is-muted is-outlined #{(params[:global]=='2') ? 'is-active' : ''}" %>
+        </div>
+    </div>
+    <% end %>
+    <div class="grid--cell">
+        <div class="button-list is-gutterless">
+            <%= link_to "active", "?global="+(params[:global]||'0')+"&filter=active", class: "button is-muted is-outlined #{(params[:filter]=='active' || params[:filter].nil?) ? 'is-active' : ''}" %>

```

Comment:
`query_url(global: params[:global] || '0', filter: 'active')` instead of string weirdness, likewise subsequent lines